.pylintrc

```diff
@@ -66,22 +66,19 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-disable=arguments-renamed,              # TODO: investigate / re-enable
-        bad-mcs-classmethod-argument,   # TODO: investigate / re-enable
+disable=arguments-renamed,              # unnecessary  
         bad-continuation, bad-whitespace, # differences of opinion with black
-        consider-using-f-string,        # TODO: investigate / re-enable
+        consider-using-f-string,        # unnecessary
         consider-using-with,            # TODO: investigate / re-enable
-        docstring-first-line-empty,     # TODO: investigate / re-enable
+        docstring-first-line-empty,     # unnecessary
         duplicate-code,                 # too verbose
         fixme,                          # avoid that to do annotations show up as warnings
-        invalid-name,                   # TODO: investigate / re-enable
-        missing-param-doc,              # TODO: investigate / re-enable (issues with kwargs docs)
+        missing-param-doc,              # unnecessary
         no-else-return,                 # relax "elif" after a clause with a return
-        no-member,                      # TODO: investigate / re-enable
+        no-member,                      # unecessary
```

Comment:
It may help future contributors (or ourselves) to include a **short** explanation why you suggest that these rules are unnecessary.